### DISCUSSION TITLE
Keep it simple, don't add to much

### DISCUSSION THREAD

**Discussion by sandstrom** [lukevella/rallly#160] [General]
This is an awesome project! 🏅 

Just some advice, don't add too much functionality. There is also a cost in maintaining, so for longevity there is some value in keeping it simple.

Also, you could hide advanced stuff behind a paywall. E.g. pay $5 one-off fee for certain advanced functionality, or have a paid cap (pay $5 for more than 50 responses).

**Comment by lukevella** [comment]
Thanks for the feedback. I don't have any concrete plans for monetization yet but I will of course need to think about that at some point.
